### PR TITLE
Feature/shuffle

### PR DESCRIPTION
**Pull request by TommyMedina**
Some changes to shuffling and temporary demonstration of multi deck use. built upon Grants branch from the same sprint.

### CODE REVIEW

File: source/CardGames.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def __eq__(self, other):
       self.value == other.value
 #######################################################################################################################
 class Deck:
-  def __init__(self):
+  def __init__(self,deck_count=1):
```
Comment (Alex-Church) [inline source/CardGames.py]: A brief description of the functionality might be helpful for later review.

File: source/CardGames.py [hunk 2]
```diff
@@ -280,12 +280,38 @@ def winnnerCheck(self, house, player):
       return 1 if houseHand <= 21 else 2
 #####################################################################################################################
 """Initializations"""
-deck = Deck()
+
+player_name = input("Username: ")
```
Comment (Alex-Church) [inline source/CardGames.py:284]: Maybe let the user know they can just type anything to continue. ex. player_name = input("Username: (type anything)   ")

Comment (TommyMedina) [reply source/CardGames.py:284]: noted

File: source/CardGames.py (hunk 1 above)
Comment (TommyMedina) [reply source/CardGames.py]: I have added a comment

**Pull request merged**